baselines/constraint/constraint_wrapper.py

```diff
@@ -70,6 +110,32 @@ def reset(self, **kwargs):
         return ob
 
     def step(self, action):
+        """
+        Perform a step in the environment and 
+        evaluate if any constraints have been violated.
+
+        Parameters
+        ----------
+        self: ConstraintEnv
+            A data structure that serves as 
+            a wrapper to the Gym environment.
+        act: ActWrapper
+            A function that takes a batch of
+            observations and returns actions.
+
+        Returns
+        -------
+        ob: Tensor?
```

Comment:
This is going to be an np.array ultimately, but that might be wrapped up in some data structure